data/en/mbs_0251.json

```diff
@@ -581,7 +581,7 @@
                             },
                             {
                                 "goto": {
-                                    "group": "summary-section"
```

Comment:
@nealedj Just tested and this change works fine. Just to confirm though, in this schema employees section comes after significant changes? as in all other employee schemas for mbs this is the opposite 😕